src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.Figure;
+
+public class Circle extends Figure {
+    private double radius;
+    private String color;
+
+    public Circle(double radius, String color) {
+        this.radius = Math.round(radius * 10.0) / 10.0;
```

Comment:
the same. implement this logic outside the constructor. You are an object of universality
check other classes